plugins/base/base-test-utils/src/main/kotlin/renderers/JsoupUtils.kt

```diff
@@ -29,6 +29,8 @@ class A(vararg matchers: Any) : Tag("a", *matchers)
 class B(vararg matchers: Any) : Tag("b", *matchers)
 class I(vararg matchers: Any) : Tag("i", *matchers)
 class STRIKE(vararg matchers: Any) : Tag("strike", *matchers)
+
+class BLOCKQUOTE(vararg matchers: Any) : Tag("blockquote", *matchers)
```

Comment:
`STRIKE` seems to be the only one in all caps (not sure why), so I'd stick to the majority
```suggestion
class Blockquote(vararg matchers: Any) : Tag("blockquote", *matchers)
```